src/api/v1beta1/properties.go

```diff
@@ -387,3 +406,30 @@ func splitArg(arg string) (key, value string) {
 	value = split[1]
 	return
 }
+
+func getRawImageTag(imageURI string) string {
+	if !strings.Contains(imageURI, ":") {
+		return "latest"
+	}
+	splitURI := strings.Split(imageURI, ":")
+	return splitURI[len(splitURI)-1]
+}
+
+// getSafeImageTag extracts and normalizes the provided image's tag
+func getSafeImageTag(imageURI string) string {
```

Comment:
```suggestion
func getSafeImageTagForLabel(imageURI string) string {
```
so it is visible for what it has been made safe